WinUIGallery/Samples/ControlPages/TitleBarPage.xaml

```diff
@@ -19,220 +19,157 @@
-                                                <Setter Property="Width" Value="{StaticResource SwatchSize}" />
-                                                <Setter Property="Height" Value="{StaticResource SwatchSize}" />
-                                                <Setter Property="RadiusX" Value="4" />
-                                                <Setter Property="RadiusY" Value="4" />
-                                            </Style>
-                                        </GridView.Resources>
-                                        <GridView.Items>
-                                            <Rectangle AutomationProperties.Name="Transparent" Fill="Transparent" />
-                                            <Rectangle AutomationProperties.Name="Red" Fill="Red" />
-                                            <Rectangle AutomationProperties.Name="Orange" Fill="Orange" />
-                                            <Rectangle AutomationProperties.Name="Yellow" Fill="Yellow" />
-                                            <Rectangle AutomationProperties.Name="Green" Fill="Green" />
-                                            <Rectangle AutomationProperties.Name="Blue" Fill="Blue" />
-                                            <Rectangle AutomationProperties.Name="White" Fill="White" />
-                                            <Rectangle AutomationProperties.Name="Black" Fill="Black" />
-                                        </GridView.Items>
-                                    </GridView>
+                <Grid
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Center"
+                    Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+                    BorderBrush="{ThemeResource SurfaceStrokeColorDefaultBrush}"
+                    BorderThickness="1"
+                    CornerRadius="{StaticResource OverlayCornerRadius}">
+                    <TitleBar
+                        Title="{Binding ElementName=TitleBox, Path=Text, Mode=OneWay}"
+                        IsBackButtonVisible="{Binding ElementName=BackButtonToggle, Path=IsOn, Mode=OneWay}"
+                        IsPaneToggleButtonVisible="{Binding ElementName=PaneToggle, Path=IsOn, Mode=OneWay}"
+                        Subtitle="{Binding ElementName=SubtitleBox, Path=Text, Mode=OneWay}">
+                        <TitleBar.IconSource>
+                            <ImageIconSource ImageSource="/Assets/Tiles/GalleryIcon.ico" />
+                        </TitleBar.IconSource>
+                        <TitleBar.RightHeader>
+                            <PersonPicture
+                                Width="30"
+                                Height="30"
+                                Initials="JD" />
+                        </TitleBar.RightHeader>
+                        <TitleBar.Content>
+                            <AutoSuggestBox
```

Comment:
For some reason, this is cut off.

Reply:
This is because of: https://github.com/microsoft/microsoft-ui-xaml/issues/10421

No workaround really.